src/components/breadcrumbs/_variables.scss

```diff
@@ -12,10 +12,11 @@
 $ouiBreadcrumbSpacing: $ouiSizeS !default;
 $ouiBreadcrumbTruncateWidth: $ouiSize * 10 !default;
 
-$ouiBreadcrumbBlueBackground: #B9D9EB !default;
-$ouiBreadcrumbGrayBackground: #D9E1E2 !default;
-$ouiBreadcrumbCollapsedLink: #002A3A !default;
-$ouiBreadCrumbHoverColor: #535861 !default;
+$ouiBreadcrumbBlueBackground: tintOrShade($ouiColorPrimary, 72%, 41%) !default;
+$ouiBreadcrumbGrayBackground: tint($ouiColorLightShade, 14%) !default;
```

Comment:
We should definitely rename these, because different themes have different primary colors. How about `$ouiBreadcrumbActiveBackgroundColor` and `$ouiBreadcrumbInactiveBackgroundColor`

Reply:
Would we consider this a breaking change? We don't list these variables in the list of Sass variables we have on the docsite, but at the same time they've gone into a release and are usable by builders

Reply:
No, not a breaking change - the doc site represents our public API, so undocumented vars are considered internal.